### DISCUSSION TITLE
Singularity Monitoring Dashboard

### DISCUSSION THREAD

**Discussion by xinaxu** [data-preservation-programs/singularity#379] [Ideas]
## Overview
Hi community, as we demonstrate the monitoring dashboard for Singularity in Iceland and Vegas, we would love to know if you have additional feedback or additional request for what should be included in the dashboard.

Here is a brief summary of what's included in the monitoring

## Exploring Prepared Dataset
The dashboard should list all prepared dataset, for each dataset preparation, it should allow the user to check for preparation status as well as prepared data

### Preparation Jobs
This is a view to display all jobs of different status. Clicking on the job will display all files included in that job

### Prepared Pieces
This is a view to display all pieces (i.e. piece_cid and piece_size). Clicking on each piece will display all files included in that piece as well as all deals made for that piece

### Path Exploration
This is a view to allow the user to click into each subfolder from the root of the dataset and check the status of each file. Clicking on each file will display the piece that contains that file as well as associated deals

### Deal making dashboard
This is a chart only dashboard which includes
* number of total active deals
* size of total active deals
* number of deals over time (per interval and/or accumulated) of different type
* filters on SP, dataset, client, status of deal
### Deals table
This is a table only view which list all deals and can filter on above fields. Clicking onto each deal will show relevant piece information and the files associated with that deal

**Comment by orvn** [comment]
Working on some design ideation (UX and UI) as a first step toward this
